DIRECTORY.md

```diff
@@ -577,6 +577,7 @@
   * [Hexagonal Number](maths/hexagonal_number.py)
   * [Integration By Simpson Approx](maths/integration_by_simpson_approx.py)
   * [Is Ip V4 Address Valid](maths/is_ip_v4_address_valid.py)
+  * [Is Palindrome](maths/is_palindrome.py)
```

Comment:
Why move this to maths?

Reply:
Not my change, automatic update of branches from a previous commit. And this is due to a previous algorithm being merged which was a palindromic algorithm for maths

Reply:
???  Should we move it back to `strings`?

Reply:
This is an automatic change to the DIRECTORY.md, it is not a mistake - #8748